src/extension.ts

```diff
@@ -13,6 +13,8 @@ export interface Settings {
 
 const decorationType = sourcegraph.app.createDecorationType && sourcegraph.app.createDecorationType()
 
+const statusBarItemType = sourcegraph.app.createStatusBarItemType()
```

Comment:
Check for `sourcegraph.app.createStatusBarItemType`
```suggestion
const statusBarItemType = sourcegraph.app.createStatusBarItemType && sourcegraph.app.createStatusBarItemType()
```